apps/atqr/api/src/app/challenge.controller.ts

```diff
@@ -157,7 +157,7 @@ export class ChallengeController {
     return {} as Challenge;
   }
 
-  private async updateStatus(id: Guid, status: ChallengeStatus): Promise<void> {
+   async updateStatus(id: Guid, status: ChallengeStatus): Promise<void> {
```

Comment:
What was the reasoning to remove the private keyword?
If your intention was to make the updateStatus method publicly available you failed to include the required
decorator to make it a endpoint. Remember that you are in the context of a controller.
Howerver, this method was intend to be private and to only contain the logic pertaing to statuses updates.
The endpoint that would be called, and later call this method is the endpoint updateChallenge (line 127).

Also, beware of formatting and linting before commiting and making PR. There's an extra space before async.
We have automated tools that check for that PRs and Commits that dont follow the style guide will no longer
be accepted by me.

Reply:
I did it because im using the updateStaus function ( of challenge.controlller ) to use in timeForTheThruth.ts on its buttons